modules/administration-guide/pages/uninstalling-che.adoc

```diff
@@ -10,36 +10,24 @@
 
 WARNING: Uninstalling {prod-short} removes all {prod-short}-related user data!
 
-To uninstall an instance of {prod} {prod-ver}:
+Use `{orch-cli}` to uninstall the {prod-short} instance.
 
 .Prerequisites
 
-* An active `{orch-cli}` session with administrative permissions to the destination {orch-name} cluster. See {orch-cli-link}.
-
 * `{prod-cli}`. See: xref:installing-the-chectl-management-tool.adoc[].
 
 .Procedure
 
-. Obtain the name of the {prod-short} {orch-namespace}. The default is `{prod-namespace}`.
-+
-[subs="+attributes,quotes"]
-----
-$ {orch-cli} get checluster --all-namespaces \
-  -o=jsonpath="{.items[*].metadata.namespace}"
-----
-
-. Remove the {prod-short} instance from the `{prod-namespace}` {orch-namespace}:
+* Remove the {prod-short} instance:
 +
 [subs="+attributes,quotes"]
 ----
-$ {prod-cli} server:delete -n {prod-namespace} # <1>
+$ {prod-cli} server:delete
 ----
-<1> The {orch-namespace} that you got in step 1.
 
-include::example$snip_{project-context}-note-uninstalling-in-minikube-cluster.adoc[]
-
-[NOTE] 
+[TIP]
 ====
-If {prod-short} was installed from the OpenShift web console, `{prod-cli}` will not uninstall the {devworkspace} operator.
-To uninstall the {devworkspace} Operator, see link:https://docs.openshift.com/container-platform/{ocp4-ver}/web_console/odc-about-web-terminal.html#deleting-the-devworkspace-operator-dependency[Deleting the {devworkspace} Operator dependency].
+The `--delete-namespace` option removes the {prod-short} namespace.
+
+The `--delete-all` option removes the {devworkspace} Operator and the related resources. The {prod-short} namespace isn't removed.
```

Comment:
`The {prod-short} namespace isn't removed.`
Because the subject is missing, this sentence might be misinterpreted as an expectation or reader to the reader.
Prob the best would be to drop this sentence because the first sentence is clear about what gets removed, or make it clear that it is this option that does not remove the namespace.